sds_data_manager/stacks/sds_data_manager_stack.py

```diff
@@ -0,0 +1,216 @@
+# Standard
```

Comment:
Can I request some docs here to just explain what this does? "data management" is a little vague. Actually in general, a lot of these files could use a 1-2 sentence at the top just explaining what it is and what it's used for.

Reply:
Done.